.github/workflows/staging_deploy.yml

```diff
@@ -16,8 +16,7 @@ on:
   
 env:
   environment: staging
-  #TODO: Update AZURE_DIGITAL_TEST -> AZURE_DIGITAL_STAGING upon permissions
-  credentials: ${{ secrets.AZURE_DIGITAL_TEST }}
+  credentials: ${{ secrets.AZURE_DIGITAL_STAGING }}
```

Comment:
do we need to update this inside the `staging_infrastructure.yml` file and `infrastructure.yml` file?

Reply:
Yep they have been updated in a separate on-going PR https://github.com/UK-Export-Finance/dtfs2/pull/948